### ISSUE TITLE
Ubuntu system restarts at loss.backward()

### ISSUE THREAD

**Issue by HaiminZhang** [hongyi-zhang/mixup#2]
Ubuntu system restarts at loss.backward().  I am using Ubuntu 16.04 and cuda 9.0. Could assistance be provided to resolve this issue? Much appreciated.

**Comment by HaiminZhang** [comment]
Sorry This issue has been solved.  It was caused by cuda configuration error.

**Comment by SHAIIM04** [comment]
> Sorry This issue has been solved. It was caused by cuda configuration error.

how are you solve . please?
